okdownload/src/main/java/com/liulishuo/okdownload/core/file/MultiPointOutputStream.java

```diff
@@ -303,7 +303,10 @@ void inspectStreamState(StreamsState state) {
         boolean isNoMoreStream = true;
         state.newNoMoreStreamBlockList.clear();
 
-        final int outputStreamCreatedBlockCount = noMoreStreamList.size();
+        @SuppressWarnings("unchecked")
+        final List<Integer> outputStreamCreatedBlockList = (List<Integer>) noMoreStreamList.clone();
```

Comment:
`outputStreamcreatedBlockList` not really equal to `noMoreStreamList`, you can use `clonedList`.